src/views/virtualmachines/details/components/details/components/grid/rightGrid/RunningVirtualMachineDetailsRightGrid.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+// import { printableVMStatus } from 'src/views/virtualmachines/utils';
```

Comment:
Is this just an oversight or did you leave it for a reason?

Reply:
+1, removed :)